src/main/java/com/mindera/pizza/domain/address/Address.java

```diff
@@ -19,71 +21,43 @@ public class Address extends DatabaseTimestamps{
     private Long id;
 
     @Getter @Setter
+    @NotBlank(message = DataValidationConstants.INVALID_STREET_NAME)
```

Comment:
temos 2 dominios de classes, "dto" e "domain"
sendo que a informação que vem do exterior "passa" primeiro pela classes dto não seria melhor alocar estas validações no objeto de entrada? se levantar incoerencias, levanta logo no início e não na fase quando já vai para guardar na BD